services/grip/frontend/src/components/CreateContact.vue

```diff
@@ -0,0 +1,97 @@
+<script setup lang="ts">
+import { ref, inject } from "vue";
+import InputText from "primevue/inputtext";
+import Button from "primevue/button";
+import { useToast } from "vue-toastification";
+import axios from "axios";
+import QRCode from "./common/QRCode.vue";
+import QrcodeVue from "qrcode.vue";
+
+// To store credentials
+const create_contact_alias = ref("");
+
+// For notifications
+const toast = useToast();
+
+// For state
+let processing = ref(false);
+let invitation_url = ref("");
+
+// Grab our store
+const store: any = inject("store");
+
+const submit_new_contact = () => {
+  processing.value = true; // Disable button while processing
+
+  // const data = `username=${key.value}&password=${secret.value}`;
+
+  axios({
+    method: "post",
+    url: "http://localhost:5100/tenant/v1/contacts/create-invitation",
```

Comment:
Hardcoded to locahost. Will have to consult with @popkinj on plan for configuring which Traction API URL the Tenant UI calls against.
I think this will have to take some thought for how we supply config on the dev/test/int/prod deployed environments. Probably for now can just do whatever suffices for local Docker development.

If there's merge or blocking issues I guess we could just merge in with hardcoded localhost and then address once environment configuration is sorted in a subsequent PR.